bsp/m5stack_core_s3/m5stack_core_s3.c

```diff
@@ -88,6 +89,13 @@ esp_err_t bsp_i2c_init(void)
     BSP_ERROR_CHECK_RETURN_ERR(i2c_master_bus_add_device(i2c_handle, &aw9523_config, &aw9523_h));
 
     i2c_initialized = true;
+
+    /*
+     * Here we explicitly enable camera, because there is no other API that would enable it.
+     * All other peripherals (touch, audio, SD card...) are enabled in their init call (eg. bsp_sdcard_mount()),
+     * but we do not have a BSP init call for camera.
+     */
+    BSP_ERROR_CHECK_RETURN_ERR(bsp_enable_feature(BSP_FEATURE_CAMERA));
```

Comment:
We can make it as public API like here: https://github.com/espressif/esp-bsp/blob/6c402325d9e2624608143ce39ead9c65bb9a7b34/bsp/m5stack_core_2/include/bsp/m5stack_core_2.h#L357C11-L357C29 
And then enable it in example or application.

Reply:
mm, but then I cannot write 1 example for multiple BSPs... we did not consider this before... Let's discuss tomorrow

Reply:
I understand. I am thinking, that there will be nice to add some define to each BSP. Something like this:
```
#define BSP_ESP_BOX_3
```

And then we can use `#ifdef BSP_ESP_BOX_3` for special API in examples. I have similar problem in benchmark example [here](https://github.com/espressif/esp-bsp/blob/244b10bb4bd598178d27e1f31a5e10b190eacf9d/examples/display_lvgl_benchmark/main/main.c#L21)